geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml

```diff
@@ -0,0 +1,64 @@
+<featureType>
+  <id>FeatureTypeInfoImpl--679c7b53:182fb30d43f:-7215</id>
+  <name>PIMS_PROPERTY_BOUNDARY_VW</name>
+  <nativeName>PIMS_PROPERTY_BOUNDARY_VW</nativeName>
+  <namespace>
+    <id>NamespaceInfoImpl-7e0faa3c:17c208de014:-7ff5</id>
+  </namespace>
+  <title>PIMS_PROPERTY_BOUNDARY_VW</title>
+  <keywords>
+    <string>features</string>
+    <string>PIMS_PROPERTY_BOUNDARY_VW</string>
+  </keywords>
+  <nativeCRS class="projected">PROJCS[&quot;NAD83 / BC Albers&quot;, 
+  GEOGCS[&quot;NAD83&quot;, 
+    DATUM[&quot;North American Datum 1983&quot;, 
+      SPHEROID[&quot;GRS 1980&quot;, 6378137.0, 298.257222101, AUTHORITY[&quot;EPSG&quot;,&quot;7019&quot;]], 
```

Comment:
Thanks for adding the new map layer to our local geoserver. Could you please also update **make env** to match these changes?

Reply:
@asanchezr my understanding of make .env is that is creates the datastore.xml files needed to connect to a database. In this case, the same datastore previously used for the location view can also be used for the boundary view (note the co-location of both views under the same parent folder).

Reply:
Right. I missed that detail. Thanks for clarifying